GVRf/Sample/gvr-test-screenshot3Dresult/src/org/gearvrf/testscreenshot3Dresult/TestScreenshot3DResultScript.java

```diff
@@ -0,0 +1,99 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.testscreenshot3Dresult;
+
+import org.gearvrf.GVRAndroidResource;
+import org.gearvrf.GVRContext;
+import org.gearvrf.testscreenshot3Dresult.R;
+import org.gearvrf.FutureWrapper;
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRScene;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRScript;
+
+import android.util.Log;
+
+public class TestScreenshot3DResultScript extends GVRScript {
+
+    private static final float CUBE_WIDTH = 20.0f;
+    private GVRContext mGVRContext = null;
+
+    @Override
+    public void onInit(GVRContext gvrContext) {
+        mGVRContext = gvrContext;
+
+        GVRScene scene = mGVRContext.getNextMainScene();
+
+        FutureWrapper<GVRMesh> future_mesh = new FutureWrapper<GVRMesh>(
```

Comment:
Same code comments as in `SampleCubeScript.java`.

These two look very similar - why are you adding two so-similar samples?